code/modules/shuttle/supply.dm

```diff
@@ -67,10 +67,9 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 /obj/docking_port/mobile/supply/proc/check_blacklist(areaInstances)
 	for(var/place in areaInstances)
 		var/area/shuttle/shuttle_area = place
-		for(var/trf in shuttle_area)
-			var/turf/T = trf
-			for(var/a in T.GetAllContents())
-				if(is_type_in_typecache(a, GLOB.blacklisted_cargo_types) && !istype(a, /obj/docking_port))
+		for(var/turf/shuttle_turf as anything in shuttle_area)
+			for(var/atom/item in shuttle_turf.GetAllContents())
```

Comment:
Might want to pick a better name, like `passenger` or something--not everything in the shuttle contents is an item (`/obj/item`).

Reply:
Done!